README.md

```diff
@@ -1,4 +1,4 @@
-[AppVeyor](https://ci.appveyor.com/project/dstebila/liboqs): ![Build status image](https://ci.appveyor.com/api/projects/status/9d2ts78x88r8wnii/branch/main?svg=true), [CircleCI](https://circleci.com/gh/open-quantum-safe/liboqs/tree/main): ![Build status image](https://circleci.com/gh/open-quantum-safe/liboqs/tree/main.svg?style=svg)
+[AppVeyor](https://ci.appveyor.com/project/dstebila/liboqs): ![Build status image](https://ci.appveyor.com/api/projects/status/9d2ts78x88r8wnii/branch/main?svg=true), [CircleCI](https://circleci.com/gh/open-quantum-safe/liboqs/tree/main): ![Build status image](https://circleci.com/gh/open-quantum-safe/liboqs/tree/main.svg?style=svg), [TravisCI](https://travis-ci.com/github/open-quantum-safe/liboqs): [![Build Status](https://travis-ci.com/open-quantum-safe/liboqs.svg?branch=main)](https://travis-ci.com/open-quantum-safe/liboqs)
```

Comment:
May I suggest reverting (removing) this addition again as a) it's failing on `main` and b) [didn't we agree that Travis (and the platforms it tests) are no "mainline" targets until they deliver the same level of testing as CCI](https://github.com/open-quantum-safe/liboqs/pull/1037#issuecomment-870345003)?

Reply:
Do you suggest to revert the travis build (travis config file), or only the Travis build icon in the readme?
To a) The build will pass once #1037 is merged
To b) My thought was to enable the simple travis build for ppc64le (on main), but don't advertise the platform as "fully supported" until full CCI is integrated, I might have misunderstood it though. In the meanwhile I see the travis build mainly as an additional test to check if libOQS builds and which can be useful to detect things like #1038.

Reply:
No, the test shall remain: I'd have argued for the icon to go -- but if it's turning green on merge of #1037 I'm fine. Begs the question what's missing for merging #1037? I thought #1049 has been created to allow us to merge #1037? Right @dstebila ?

Reply:
> No, the test shall remain: I'd have argued for the icon to go -- but if it's turning green on merge of #1037 I'm fine. Begs the question what's missing for merging #1037? I thought #1049 has been created to allow us to merge #1037? Right @dstebila ?

I'd just been waiting for the builds to finish on #1037, which they now have, so I've merged #1037.  We might as well keep the Travis icon since it's there, even though we're not officially advertising support for ppc64le yet.  And for our dev team, you can always see the full table of all our build icons at https://openquantumsafe.org/dashboard.html.